### ISSUE TITLE
StickyToolbar component

### ISSUE THREAD

**Issue by kueda** [inaturalist/iNaturalistReactNative#484]
This is a toolbar that sticks to the bottom of the screen and renders children, e.g.

```javascript
<StickyToolbar>
  <Button text="Foo" />
  <Button text="Bar" />
</StickyToolbar>
```

I think the only props need to be its children

Requirements

- [ ] Sticks to the bottom of the screen, absolutely positioned
- [ ] Casts a shadow, something like `box-shadow: 0px -2px 2px rgba(0, 0, 0, 0.25);`
- [ ] 15px of padding

Example:

<img width="758" alt="Screen Shot 2023-02-13 at 5 38 35 PM" src="https://user-images.githubusercontent.com/22691/218615922-14e88e9b-019b-422d-8e6c-b20c09b2e362.png">

**Comment by Chrischuck** [comment]
I can get a pr up for this

**Comment by albullington** [comment]
Closed by #506
